pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -77,36 +93,34 @@ func NewNodeController(
 	}
 }
 
-// Run creates initial node list and start syncing instances from cloudprovider if any.
+// Run creates initial node list and start syncing instances from cloudprovider, if any.
 // It also starts syncing cluster node status.
 // 1. RegisterNodes() is called only once to register all initial nodes (from cloudprovider
 //    or from command line flag). To make cluster bootstrap faster, node controller populates
 //    node addresses.
-// 2. SyncCloud() is called periodically (if enabled) to sync instances from cloudprovider.
+// 2. SyncCloudNodes() is called periodically (if enabled) to sync instances from cloudprovider.
 //    Node created here will only have specs.
 // 3. SyncNodeStatus() is called periodically (if enabled) to sync node status for nodes in
```

Comment:
Maybe you should call it ProbeNodeStatus() [or SyncProbedNodeStatus()] instead of SyncNodeStatus(), now that there are really two ways of syncing the node status.